### PR TITLE
fix(style): single row items are too tall

### LINKED ISSUE #297: Single row items are too tall
screen shot 2015-02-06 at 10 20 18 am

These should ~10px of padding top and bottom rather than ~20px.

### PR DESCRIPTION
**Pull request by johngruen**
Fixes #297

### CODE REVIEW

**Comment by johngruen** [comment]
@pdehaan: not sure why that dump.rdb file is hanging around...added it to my global .gitignore, but it's not being very polite about not showing up in the commit as a deleted.

**Comment by johngruen** [comment]
#297

File: app/styles/modules/_user_page.scss [hunk 1]
```diff
@@ -103,10 +110,15 @@
...
       margin: 0;
+      padding-right: $default-gap;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      white-space: nowrap;
     }
   }
 
   // Fancy sibling selectors to achieve groupings
+  // scss-lint:disable MergeableSelector
```
Comment (pdehaan) [inline app/styles/modules/_user_page.scss]: Nice trick! :clap:

**Comment by pdehaan** [comment]
chronicle
**Before:** .small internal height is 104px.

chronicle
**After:** .small internal height is 76px.

**Pull request merged**